_data/social.yml

```diff
@@ -89,6 +89,12 @@ websites:
       img: pinterest.png
       tfa: No
 
+    - name: Quora
+      url: https://www.quora.com
+      img: quora.png
+      tfa: No
+      twitter: quora      
```

Comment:
there's a tab at the end of the twitter handle. That could mess up the yml file.